core/includes/common.inc

```diff
@@ -8407,6 +8407,10 @@ function backdrop_implode_tags($tags) {
  * invokes a hook so that other modules' cache data can be cleared as well.
  */
 function backdrop_flush_all_caches() {
+  // Rebuild all color palettes.
+  if (module_exists('color')) {
+    color_rebuild_settings();
+  }
```

Comment:
Instead of hard-coding color module into `backdrop_flush_all_caches()` could we instead use `hook_flush_caches()` and put it in `color.module`?